models/classes/LtiLaunchData.php

```diff
@@ -550,7 +560,9 @@ public function getReturnUrl()
     public function jsonSerialize()
     {
         return [
-            'variables' => $this->variables,
+            'variables' => array_map(function ($var) {
+                return $var instanceof AgsClaim ? $var->normalize() : $var;
+            }, $this->variables),
```

Comment:
This is not good place for this operation. Should we extract it to some method instead?

Reply:
Changed callback to arrow fn for better readability as @andreluizmachado proposed.
Anyway for me it looks much more transparent then moved out somewhere